xpo.obo

```diff
@@ -57485,7 +57485,7 @@ name: increased size of the dorsal region
 name: obsolete increased size of the dorsal region
 def: "An increase in the size of the dorsal region." []
 synonym: "enlarged dorsal region" EXACT []
-is_a: XPO:0107363 ! abnormal dorsal region morphology
+is_a: XPO:0107363 ! obsolete abnormal morphology of dorsal region
```

Comment:
@seger can you explore what the reason for this weird thing is? Did we obsolete XPO:0107363 and forgot to remove it from somewhere again?